### PR TITLE
New theme

### PR DESCRIPTION
**Pull request by atesgoral**
Also fixed issue with agency field not being enabled once it has been disabled

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -585,86 +561,64 @@
...
-            <div class="quick-col form-group">
-              <label for="databaseID">
-                Database
-              </label>
-              <br>
-              <input type="checkbox" class="checkbox" id="databaseID" name="databaseID" value="Yes"/>
+          </fieldset>
+          <fieldset>
+            <div class="_checkbox-wrapper">
+              <input type="checkbox" id="databaseID"> <label for="databaseID">Database</label>
```
Comment (sonalranjit) [inline popup.html:603]: Can we add a Value tag to all the Toggle buttons to be "Yes", just to be consistent with the previous submissions, currently it submits "on". Besides that looks great

**Review by sonalranjit — CHANGES_REQUESTED** [review]
Everything looks good, just a minor change request for the FTP, Visualization, Database and Many File Toggle buttons are submitting "on". Maybe just add back the `Value = "Yes"` tag just to keep the submissions consistent.

File: popup.html (hunk 1 above)
Comment (atesgoral) [reply popup.html:603]: Oops! Realized that last night and forgot about it :) Doing it now.

Comment (atesgoral) [reply popup.html:603]: OK. Did it in an intentionally roundabout way for decoupling of collected data from sheet representation.

**Review by sonalranjit — APPROVED** [review]

**Comment by sonalranjit** [comment]
looks good

**Pull request merged**